cmd/config/docs/api-conventions/functions-spec.md

```diff
@@ -290,6 +308,10 @@ results:
       name: wordpress
     field:
       path: spec.ports.0.port
+      currentValue: cool-port
+      proposedValues:
+      - 1337
+      - 4242
```

Comment:
To be consistent with the indentation above, please add 2 more whitespaces here.